src/test/java/com/sfuentes/CarLicensePlateTest.java

```diff
@@ -0,0 +1,109 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CarLicensePlateTest {
+
+  @Test
+  void getCountOfEachColorByLicensePlate_FiveLicensePlate_OneOfEachColor() {
+    String[] licensePlates = {"XX-55-90", "650-008", "AA3-76-KL", "950-L0-14", "ALT-L80-2"};
+    CarLicensePlate test = new CarLicensePlate(licensePlates);
+    Map<String, Integer> expected = new HashMap<>();
+    expected.put("1 or 2 - Yellow ", 1);
+    expected.put("3 or 4 - Pink ", 1);
+    expected.put("1 or 2 - Red ", 1);
+    expected.put("3 or 4 - Green ", 1);
+    expected.put("3 or 4 - Blue ", 1);
+
+    assertEquals(expected, test.getCountOfEachColorByLicensePlate(licensePlates));
+  }
+
+  @Test
+  void getCountOfEachColorByLicensePlate_FiveLicensePlate_OneYellowColor() {
```

Comment:
From this test onward you use this pattern
 "OneYellowColor" , "OnePinkColor", "OneRedColor" etc.
I guess you wanted to say OnlyYellow because the cound of the color is qlways more than one

Reply:
fixed